experiments/ClimaCore/sea-ice/slab_ice.jl

```diff
@@ -0,0 +1,341 @@
+# # Heat Equation + Slab Tutorial
+
+#  - load external packages:
+import LinearAlgebra, UnPack
+import ClimaCore: Fields, Domains, Topologies, Meshes, DataLayouts, Operators, Geometry, Spaces
+
+using Base: show_supertypes
+using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
+
+using Logging: global_logger
+using TerminalLoggers: TerminalLogger
+
+using RecursiveArrayTools
+
+using OrdinaryDiffEq
+
+using Statistics
+
+#src ## Setup Logging Information
+global_logger(TerminalLogger())
+const CI = !isnothing(get(ENV, "CI", nothing))
+
+# ## Define Parameters
+#  - Global Constants
+const FT = Float64;
+
+#  - Experiment-specific Parameters
+parameters = (
+    ## atmos parameters
+    zmin_atm = FT(0.0), # height of atm stack bottom [m] 
+    zmax_atm = FT(1.0), # height of atm stack top [m]
+    n = 15,  # number of elements in atm stack 
+    μ = FT(0.0001), # diffusion coefficient [m^2 / s]
+    T_top = FT(280.0), # fixed temperature at the top of the domain_atm [K]
+    ## slab ice parameters
+    ρc_ml = FT(4e6), # for mixed layer
+    F_basal = FT(120), # W / m2 / K
```

Comment:
this is minor, but F_basal for the coefficient and F_base for the flux is a little confusing. maybe even a letter besides "F" for the coefficient would be helpful, since "F" denotes a flux everywhere else?

Reply:
Yeah, I agree. Great point!